steam_shortcuts.py

```diff
@@ -232,16 +233,15 @@ def get_library_folders(steam_path, library_index_path):
     Returns the library locations
     """
 
-    locations = []
+    libraries = []
 
-    # Find the lines matching a library folder (the library index, and the location)
-    p = re.compile('"\d{1,3}".+".+"')
-    with open(library_index_path.resolve(), encoding="utf-8") as index:
-        locations = [p.findall(line.strip()) for line in index if p.search(line)]
+    libVDF = vdf.load(open(library_index_path))
+
+    for lib in libVDF['libraryfolders'].values():
```

Comment:
Just in case this key is missing:

```suggestion
    for lib in lib_vdf.get("libraryfolders", {}).values():
```